packages/object-prototype/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "object-prototype",
+  "version": "3.0.2",
+  "description": "A replacement prototype for `Object.prototype` with all the same functions",
+  "main": "index.js",
+  "dependencies": {
+    "object-prototype-functions": "1.2.0"
+  },
+  "devDependencies": {
+    "debug": "^4.1.1",
+    "tape": "^4.13.0"
+  },
+  "scripts": {
+    "test": "node test.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/watson/object-prototype.git"
+  },
+  "keywords": [
+    "object",
+    "prototype",
+    "pollution",
+    "polyfill",
+    "security"
+  ],
+  "author": "Thomas Watson <w@tson.dk> (https://twitter.com/wa7son)",
+  "license": "apache-2.0",
```

Comment:
@kobelb I changed the license from `MIT` to `apache-2.0` as I guess that's easier when it's inside this repo right? It just requires a major bump next time it's published to npm.

Reply:
Correct, we use the apache-2 license for the OSS code unless explicitly denoted: https://github.com/elastic/kibana/blob/master/LICENSE.txt